homework-submission/myapp4/src/Components/Form.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { inject, observer } from "mobx-react";
+
+@inject("TodosStore")
+@observer
+
+class Form extends React.Component {
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    const descriptionInput = this.descriptionInput.value;
+    const deadlineInput = this.deadlineInput.value;
+
+    this.props.TodosStore.createTodo({
+      description: descriptionInput,
+      deadline: deadlineInput,
+      __v: 0, done: false
```

Comment:
The`__v` is a server data base versioning as I think, you don't have to worry about it, it's auto generated 🙄